bundles/AdminBundle/src/Controller/Admin/DataObject/DataObjectController.php

```diff
@@ -552,158 +552,6 @@ static function (Task $task) {
         throw $this->createAccessDeniedHttpException();
     }
 
-    /**
```

Comment:
that means the action `pimcore_admin_dataobject_dataobject_relation_objects_list` is also not available anymore ... which is at least used in `bundles/AdminBundle/public/js/pimcore/object/tags/manyToOneRelation.js` ... I think there we have to think about another facade mechanism or something...

Reply:
We could do the same as for `manyToManyRelation` - simply hiding the `combo` option if no search implementation has been registered. See https://github.com/pimcore/pimcore/blob/4ea76185670345f7a9d89554c20ada9d93ae16c2/bundles/AdminBundle/public/js/pimcore/object/classes/data/advancedManyToManyObjectRelation.js#L129-L134

Reply:
implemented another facade method `getObjectRelationInlineSearchRoute`. See 78d4ae2c9447daff4f83173c6d38ce5f2073ea64